test/lib/Router_test.js

```diff
@@ -1113,11 +1119,14 @@ describe('API Router', function() {
             this.router.route('/derp', {
                 type: 'object',
                 properties: {
-                    flerp: { type: 'string' }
+                    flerp: { type: 'string' },
+                    fleep: { type: 'string' }
```

Comment:
@marianeaj Added extra property to the /derp route that isn't being set in the tests that show successful validation - so yeah, that works.